examples/discord_bot.py

```diff
@@ -52,18 +52,47 @@ async def player_heroes(ctx, player_tag):
 async def clan_info(ctx, clan_tag):
     clan = await coc_client.get_clan(clan_tag)
 
+    for a in clan.members:
+        if a.role == a.role.leader:
+            leader = a
+    if clan.public_war_log is False:
+        log = 'Private'
+    else:
+        log = 'Public'
     e = discord.Embed(colour=discord.Colour.green())
     e.set_thumbnail(url=clan.badge.url)
-    e.add_field(name=clan.name, value=clan.tag)
-    e.add_field(name="Description", value=clan.description)
-    e.add_field(name="Members", value=", ".join([member.name for member in clan.members]))
-    e.add_field(
-        name="Clan Record", value="{}-{}-{}".format(clan.war_wins, clan.war_losses, clan.war_ties),
-    )
-
+    e.add_field(name='Clan Name', value=f'{clan.name}({clan.tag})\n[Open in game]({clan.share_link})', inline=False)
+    e.add_field(name='Clan Level', value=clan.level, inline=False)
+    e.add_field(name="Description", value=clan.description, inline=False)
+    e.add_field(name="Leader", value=f'{leader}', inline=False)
```

Comment:
Consistent use of just passing the object (ie `e.add_field(name="Leader", value=leader, inline=False`) or passing a string of the object, ie `e.add_field(name="Leader", value=str(leader), inline=False)` would be good, however this f-string unnecessary because it's just stringifying the leader.

Reply:
Ok I will correct these